docs/sidebars.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   someSidebar: {
-    'Getting Started': ["intro", "hardware"],
+    'Getting Started': ["intro", "hardware", "faq"],
```

Comment:
I would put this before Hardware, especially given that doc isn't as refined. Thoughts?

Reply:
sounds good!